source/_integrations/mammotion.markdown

```diff
@@ -0,0 +1,65 @@
+---
+title: Mammotion
+description: Instructions on how to integrate Mammotion robotic lawn mowers with Home Assistant.
+ha_category:
+  - Lawn Mower
+ha_release: 2026.01
+ha_iot_class: Cloud Push
+ha_config_flow: true
+ha_codeowners:
+  - '@mikey0000'
+ha_domain: mammotion
+ha_platforms:
+  - lawn_mower
+ha_integration_type: integration
+ha_quality_scale: bronze
+---
+
+The **Mammotion** {% term integration %} provides BLE and Wi-Fi connectivity with Mammotion products from Home Assistant.
+
+## Prerequisites
+
+Mammotion logins only allow one device to be logged in at a time. To use this integration, you need to create a separate account just for Home Assistant and share devices with that account.
+
+To create the separate account:
+
+1. Sign up using the Mammotion App with an email address you plan to use just with Home Assistant. SSO, such as Google, is not officially supported, though Google email addresses do work.
+2. Sign in to your main account and share the devices you want to control with the new account.
+3. Sign in to the new account and accept the shared devices.
+
+{% include integrations/config_flow.md %}
+
+Use your new account credentials during setup:
+
+{% configuration_basic %}
+Email:
+    description: Your Mammotion account email
+Password:
+    description: Your Mammotion account password
+{% endconfiguration_basic %}
+
+## Lawn mower
+
+The integration creates a lawn mower entity to control the mower. This entity can:
+
+- Pause/Resume mowing
+- Return to dock
+- Status monitoring
+
+
+## Known issues
+
+Because Home Assistant now talks to your device, this can sometimes cause issues with making changes using the Mammotion App, though the integration will stop communicating when it sees the device is in setup states, the device will continue to work as expected even if you see a delay visually on your phone when setting up zones, areas or channels. 
```

Comment:
```suggestion
Because Home Assistant now talks to your device, this can sometimes cause issues with making changes using the Mammotion App. The integration stops communicating when it sees the device is in setup state. However, the device continues to work in the background, even if you notice a delay on your phone when setting up zones, areas, or channels. 
```
Not sure if I fully understood this section. Tried to rephrase it a bit. Please feel free to change it if the edited version is incorrect.